### DISCUSSION TITLE
Allow APP_NAME environment variable to override .env file.

### DISCUSSION THREAD

**Discussion by ChaoticWyrme** [linuxserver/Heimdall#734] [Ideas]
I'm hosting my heimdall install in docker, and wanted to change the tab title. I followed the APP_NAME environment variable from #132, by adding -e APP_NAME="Media Server" onto my docker run command. However, this didn't change the tab title, and looking at the filesystem in the container, it is specified in the .env file, so it may be that it is being overridden by that.

I would like to be able to specify the APP_NAME setting through my `docker run` command rather than modifying the file in the container in such a fragile way.

**Comment by BenAhrdt** [comment]
I also like to change the name of the Tab.
Just haveing 2 or more containers of Heimdall.
1 for "Smarthome"
1 for "Private things"
2 for "..."

**Comment by bardspaun** [comment]
Any updates on this?

**Comment by andymarden** [comment]
I tried the APP_NAME thing which does not work for me.

It's actually dead easy though - go to Settings -> Advanced and paste the following into Custom Javascript:
```
document.title = "Your New Title Here";
```
It changes the browser tab immediately.

Pretty sure you could add an event listener this way and get it to change the title when different things occurred,.
